### DISCUSSION TITLE
yarn build构建的时候报错，webpack的原因，真的不知道怎么解决，在线求助，node18.16

### DISCUSSION THREAD

**Discussion by CXTCodes** [ChatGPTNextWeb/NextChat#1385] [Q&A]
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/42209442/6523dbbd-1044-4366-904d-aa23101a15d8)

**Comment by Yidadaa** [comment]
没有遇到过该错误，看起来是 page.tsx 文件出了问题，建议删掉仓库重新 pull
